src/main/java/com/example/onlinebookstore/controller/BookController.java

```diff
@@ -16,36 +20,41 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "/books")
 public class BookController {
     private final BookService bookService;
-    
+
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "get all Book", description = "get all the books")
```

Comment:
```suggestion
    @Operation(summary = "Get all Books", description = "Get all the books")
```
Same issue in other `@Operation's`